packages/integrations/netlify/test/edge-functions/fixtures/edge-basic/.netlify/edge-functions/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+  "functions": [
+    {
+      "function": "entry",
+      "path": "/"
+    },
+    {
+      "function": "entry",
+      "path": "/two"
+    }
+  ],
+  "version": 1
+}
```

Comment:
nit: did you mean to commit `.netlify` here?

Reply:
Oops, no it should be ignored. Will update.